src/configurationProvider.ts

```diff
@@ -100,11 +102,56 @@ export class CSharpConfigurationProvider implements vscode.DebugConfigurationPro
         });
     }
 
+    /**
+     * Parse envFile and add to config.env
+     */
+    private parseEnvFile(envFile: string, config: vscode.DebugConfiguration): vscode.DebugConfiguration {
+        if(envFile) {
+            try {
+                const parsedFile = ParsedEnvironmentFile.CreateFromFile(envFile, config["env"]);
+                
+                // show error message if single lines cannot get parsed
+                if (parsedFile.Warning) {
+                    CSharpConfigurationProvider.showFileWarningAsync(parsedFile.Warning, envFile);
+                }
+
+                config.env = parsedFile.Env;
+            }
+            catch (e) {
+                throw new Error("Can't parse envFile " + envFile);
+            }
+        }
+
+        // remove envFile from config after parsing
+        if(config.envFile) {
+            delete config.envFile;
+        }
+
+        return config;
+    }
+
     /**
 	 * Try to add all missing attributes to the debug configuration being launched.
 	 */
     resolveDebugConfiguration(folder: vscode.WorkspaceFolder | undefined, config: vscode.DebugConfiguration, token?: vscode.CancellationToken): vscode.ProviderResult<vscode.DebugConfiguration> {
         // vsdbg does the error checking
```

Comment:
move to above line 144 `return config` and change to something like `// vsdbg will error check the debug configuration fields`